pyomo/contrib/pynumero/examples/structured/tests/test_nlp_transformations.py

```diff
@@ -12,7 +12,8 @@
 import pyomo.environ as aml
 import os
 
-from .. import numpy_available, scipy_available
+#from .. import numpy_available, scipy_available
```

Comment:
This shouldn't be here. These changes are not part of this PR, so revert them to original state.